src/excmds.ts

```diff
@@ -2736,6 +2736,43 @@ export async function sleep(time_ms: number) {
     await new Promise(resolve => setTimeout(resolve, time_ms))
 }
 
+const promptPromises = {}
```

Comment:
Could you split this up into a separate PR? I can see that it might be useful someday but it's clearly not as --- pressing :sunglasses: --- as the gamepad support.